frontend/apps/marketing/src/components/paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,62 @@
+import '@code-dot-org/component-library/typography/index.css';
+import {
+  default as Typography,
+  StrongText,
+  VisualAppearance,
+} from '@code-dot-org/component-library/typography';
+import React, {ReactNode} from 'react';
+
+type ParagraphVisualAppearance = Exclude<
```

Comment:
Rather than using `Exclude`, could we use `Pick` for only the ones we want? If we add a new one it may show up unexpectedly.

Reply:
Thank you. Pick won't work here. We need to use Extract instead